### PR TITLE
Add Espectro ISO layout

### PR DESCRIPTION
**Pull request by WillFour20**
Add ISO layout for the Espectro keyboard.

### CODE REVIEW

File: keyboards/espectro/keymaps/iso/keymap.c [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#include QMK_KEYBOARD_H
+#define MODS_CTRL_MASK  (MOD_BIT(KC_LSHIFT)|MOD_BIT(KC_RSHIFT))
```
Comment (noroadsleft) [inline keyboards/espectro/keymaps/iso/keymap.c]: This line can be deleted; the keymap isn't using it.

**Comment by noroadsleft** [comment]
I noticed that you're filing this PR from your fork's `master` branch. While this isn't a major issue now, later changes may result in merge conflicts if you try to file another PR because the commit history will have diverged between your master and QMK's. It is highly recommended for QMK development - regardless of what is being done or where - to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your master updated here:

[**Best Practices: Your fork's master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_best_practices?id=your-fork39s-master-update-often-commit-never)

If you need any help with this just ask.

File: keyboards/espectro/keymaps/iso/keymap.c (hunk 1 above)
Comment (WillFour20) [reply keyboards/espectro/keymaps/iso/keymap.c]: Deleted

**Comment by WillFour20** [comment]
Ok I know that for the future, but I cannot change the branch for this PR and I would have to open a new one if I understand correctly. Is that really necessary for this small change?

**Comment by noroadsleft** [comment]
Sorry, I didn't clarify. You do not need to close this PR or submit a new one from a branch. It's fine for this PR; keep using the master branch if you need to update this pull request (though I don't expect you'll need to).

QMK recommends using branches and leaving the master alone because reviewing a PR that was issued from a diverged master can be challenging, because a lot of the changes will be noise. A user might submit an initial PR with a dozen commits. Not too bad. But then their PR gets merged, and then a few months later they open another PR from the diverged master with another series of commits. The first twelve commits are irrelevant; those were already merged in the first PR. That's the situation we're trying to avoid.

I myself was guilty of this in my early days with QMK and Git.

**Pull request merged**